packages/manifold/contracts/edition/ManifoldERC721Edition.sol

```diff
@@ -29,101 +29,108 @@ contract ManifoldERC721Edition is CreatorExtension, ICreatorExtensionTokenURI, I
     mapping(address => mapping(uint256 => uint256)) _maxSupply;
```

Comment:
Whats the point of `maxSupply` if the mint function is gated by the creator only?

Reply:
Hmm.. I think it's helpful when they want say 300 tokens, so you can split into 2 transactions. `maxSupply` will be 300 and then `totalSupply` will be 200 after first tx and 300 after the next tx.

But yeah - not sure why we had this in the original contract..

Reply:
ah yea make sense for multi transactions due to gas limit

Reply:
Ok, with the v3 changes as discussed, this should be changed.

struct EditionInfo {
  uint16 contractVersion;
  uint80 totalSupply;
  uint80 maxSupply;
}

mapping(address => mapping(uint256 => EditionInfo)) _editionInfo;


Now, we can store maxSupply, totalSupply and contract version into a single storage slot, further reducing memory usage.

Reply:
Also introduce:
    uint256 internal constant MAX_UINT_56 = 0xffffffffffffff;

and ensure instanceId's <= uint56 (for data packing)